networking/bgp.md

```diff
@@ -140,6 +128,56 @@ spec:
   peerSelector: route-reflector == 'true'
 ```
 
+>**Note**: Adding `routeReflectorClusterID` to a node spec will remove it from the node-to-node mesh immediately, tearing down the
+>          existing BGP sessions. Adding the BGP peering will bring up new BGP sessions. This will cause a short (about 2 seconds)
+>          disruption to dataplane traffic of workloads running in the nodes where this happens. To avoid this, make sure no
+>          workloads are running on the nodes, by provisioning new nodes or by running `kubectl drain` on the node (which may
+>          itself cause a disruption as workloads are drained).
+{: .alert .alert-info}
+
+#### Disable the default BGP node-to-node mesh
+
+The default **node-to-node BGP mesh** may be turned off to enable other BGP topologies. To do this, modify the default **BGP configuration** resource.
+
+Run the following command to disable the BGP full-mesh:
+
+```
+calicoctl patch bgpconfiguration default -p '{"spec": {"nodeToNodeMeshEnabled": false}}'
+```
+
+>**Note**: If the default BGP configuration resource does not exist, you need to create it first. See [BGP configuration]({{ site.baseurl }}/reference/resources/bgpconfig) for more information.
+{: .alert .alert-info}
+
+>**Note**: Always configure replacement BGP peerings (e.g. route reflectors) before disabling node-to-node mesh. Depending on cluster
+>          configuration, disabling the node-to-node mesh may remove all routes (by tearing down all BGP sessions), rendering the cluster's
+>          network completely unavailable. See the [BGP configuration section of the "Calico the hard way" guide]({{ site.baseurl }}/getting-started/kubernetes/hardway/configure-bgp-peering)
+>          for a more detailed step by step guide on replacing the node-to-node mesh with route reflectors.
+{: .alert .alert-danger}
+
+#### Change from node-to-node mesh to route reflectors without any traffic disruption
+
+Switching from node-to-node BGP mesh to BGP route reflectors involves tearing down BGP sessions and bringing up new ones. This causes a short
+dataplane network disruption (of about 2 seconds) for workloads running on the nodes where this happens. In order to avoid this, you may
```

Comment:
Can you be more precise about "nodes where this happens"?  The mesh affects all nodes, so do you mean "on all nodes"?

Reply:
I guess I was wrong in thinking this would affect only nodes that would be route reflectors... Yes this would be "all nodes", I'll fix it, thanks!